### PR TITLE
uncommented planners that are working

### PR DESCRIPTION
**Pull request by beatrizleon**
Only TRRTkConfigDefault is giving errors so it is still commented

### CODE REVIEW

**Comment by ugocupcic** [comment]
Not directly related to this but I'm getting the following error when loading moveit:

```
[ERROR] [1433504838.775942782]: Link 'rh_mfmiddle' is not known to URDF. Cannot disable collisons.
[ERROR] [1433504838.776000388]: Link 'rh_ffmiddle' is not known to URDF. Cannot disable collisons.
[ERROR] [1433504838.776027476]: Link 'rh_ffmiddle' is not known to URDF. Cannot disable collisons.
[ERROR] [1433504838.776063101]: Link 'rh_ffdistal' is not known to URDF. Cannot disable collisons.
[ERROR] [1433504838.776098811]: Link 'rh_rfmiddle' is not known to URDF. Cannot disable collisons.
[ERROR] [1433504838.776129159]: Link 'rh_mfmiddle' is not known to URDF. Cannot disable collisons.
[ERROR] [1433504838.776157383]: Link 'rh_mfmiddle' is not known to URDF. Cannot disable collisons.
[ERROR] [1433504838.776190388]: Link 'rh_mfdistal' is not known to URDF. Cannot disable collisons.
[ERROR] [1433504838.776223425]: Link 'rh_lfmiddle' is not known to URDF. Cannot disable collisons.
[ERROR] [1433504838.776253053]: Link 'rh_rfmiddle' is not known to URDF. Cannot disable collisons.
[ERROR] [1433504838.776282747]: Link 'rh_rfmiddle' is not known to URDF. Cannot disable collisons.
[ERROR] [1433504838.776315468]: Link 'rh_rfdistal' is not known to URDF. Cannot disable collisons.
```

**Comment by toliver** [comment]
This is my fault. I added these "disable collision" to left hand SRDF as they were necessary, and then had the brilliant idea of copying them to the right hand, But the right hand is currently biotac, so those links don't exist. I will create an issue about that.

**Comment by ugocupcic** [comment]
Should we set limited to true by default?

**Comment by ugocupcic** [comment]
@beatrizleon don't hesitate to use images and gifs in your pr, it illustrates nicely :)

screencast

**Comment by beatrizleon** [comment]
I think we should set it to true by default at least until we manage to find out a way to remove the limitation. Should I change it?

**Comment by ugocupcic** [comment]
nod

File: sr_multi_moveit/right_sr_ur10_moveit_config/config/ompl_planning.yaml [hunk 1]
```diff
@@ -6,17 +6,17 @@ planner_configs:
     type: geometric::EST
     range: 0.0  # Max motion added to tree. ==> maxDistance_ default: 0.0, if 0.0 setup()
     goal_bias: 0.05  # When close to goal select goal, with this probability. default: 0.05
-  #LBKPIECEkConfigDefault:
```
Comment (ugocupcic) [inline sr_multi_moveit/right_sr_ur10_moveit_config/config/ompl_planning.yaml:9]: shouldn't the equivalent for the left_sr_ur10... be uncommented as well?

Comment (beatrizleon) [inline sr_multi_moveit/right_sr_ur10_moveit_config/config/ompl_planning.yaml:9]: yes, you are right. I commit it as well now

**Pull request merged**